### PR TITLE
IRIS-4081 | Refactor Notifications CSS so that it can be extracted to Design System

### PR DESCRIPTION
**Pull request by slayful**
## Links

* http://wikia-inc.atlassian.net/browse/IRIS-4081

### CODE REVIEW

File: front/main/app/styles/component/_wikia-user-profile.scss [hunk 1]
```diff
@@ -36,36 +7,43 @@
...
+	&__header-wrapper {
+		.user-avatar {
+			background-color: $wds-color-white;
+			border-radius: 100%;
+			margin-top: -25px;
+			// hack for making it get on top of the header
+			transform: translate(0, 0);
+
+			figure.avatar {
+				svg,a>img  {
```
Comment (latata) [inline front/main/app/styles/component/_wikia-user-profile.scss]: space

File: front/main/app/components/notification-card.js [hunk 2]
```diff
@@ -108,7 +108,8 @@ export default Component.extend(
 
 		getTranslatedMessage(key, context) {
 			const fullContext = $.extend({}, {
-				lng: this.get('userLanguage'),
+				// TODO: XW-1685 fix i18n for User's language
```
Comment (latata) [inline front/main/app/components/notification-card.js]: remove this comment

File: front/main/app/styles/component/_wikia-user-profile.scss [hunk 3]
```diff
@@ -36,36 +7,43 @@
...
-.user-profile-fixed-wrapper {
-	flex-grow: 0;
-}
+	&__header-wrapper {
+		.user-avatar {
+			background-color: $wds-color-white;
+			border-radius: 100%;
+			margin-top: -25px;
+			// hack for making it get on top of the header
+			transform: translate(0, 0);
```
Comment (latata) [inline front/main/app/styles/component/_wikia-user-profile.scss]: `position: relative;` and `z-index: 1`

File: front/main/app/styles/component/_wikia-user-profile.scss [hunk 4]
```diff
@@ -36,36 +7,43 @@
...
 	}
-}
 
-.user-profile-fixed-wrapper {
-	flex-grow: 0;
-}
+	&__header-wrapper {
+		.user-avatar {
+			background-color: $wds-color-white;
+			border-radius: 100%;
```
Comment (latata) [inline front/main/app/styles/component/_wikia-user-profile.scss]: `border-radius: 50%` should work

File: front/main/app/styles/component/_wikia-user-profile.scss [hunk 5]
```diff
@@ -1,33 +1,4 @@
...
-				border-width: 3px;
-				height: 94px;
-				width: 94px;
-			}
-		}
-	}
-}
-
-.wikia-user-profile {
+.wds-user-profile {
```
Comment (kvas-damian) [inline front/main/app/styles/component/_wikia-user-profile.scss]: why is it prefixed with `wds` is it part of DS?

File: server/app/views/_partials/wds-spinner.hbs [hunk 6]
```diff
@@ -0,0 +1,11 @@
+<!--size = full diameter = r * 2 + stroke width-->
```
Comment (kvas-damian) [inline server/app/views/_partials/wds-spinner.hbs]: Please use comments that are not included in output file

**Comment by kvas-damian** [comment]
I'm not sure what do we want to achieve by this PR. I see many class names being prefixed with `wds-` but the're still not there.
Why cannot we move it to DS, release new version, include it in Mercury, profit?
This way we'd need to CR and test this code 2 times.

**Comment by slayful** [comment]
Well in opposition to your suggestion some people, like @latata, are against moving the CSS to Design System, if JS is not also there.

**Comment by kvas-damian** [comment]
@latata why do you think so?

**Comment by latata** [comment]
@kvas-damian Instead of having this functionality code in three places (App, Mercury, Mobile-wiki) we will have it in four places (+DS)

**Comment by kvas-damian** [comment]
and by functionality you mean CSS or something else?
I believe that after we move CSS to DS then it'll live in one place.

**Pull request merged**